src/lib/services/api-token-service.ts

```diff
@@ -88,27 +79,23 @@ export class ApiTokenService {
         }
     }
 
-    async fetchActiveTokens(): Promise<void> {
+    fetchActiveTokens = async (): Promise<void> => {
         try {
```

Comment:
I suggest that the scheduler will take the responsibility to run background task in a try/catch. I cannot think of any scenarios where we would allow a background job to crash Unleash with an unhanded exceptions.

Reply:
Good idea. I will handle both sync and async handlers. Also, I'd log those to the logger.